pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -752,20 +752,33 @@ func (r *ReconcilePerconaServerMongoDB) createOrUpdateSystemUsers(ctx context.Co
 
 	if cr.CompareVersion("1.12.0") >= 0 {
 		err = r.createOrUpdateSystemRoles(ctx, cli, "explainRole",
-			[]mongo.RolePrivilege{{
-				Resource: map[string]interface{}{
-					"db":         "",
-					"collection": "system.profile",
+			[]mongo.RolePrivilege{
```

Comment:
we need to add/change role only for >= 1.15

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1317/commits/a6659640556a5d3aa474e4f75ee9a95a37418b3b